docker/.dockerignore

```diff
@@ -0,0 +1,2 @@
+../node_modules
```

Comment:
The dockerignore file is out of context for `Dockerfile`

Reply:
Put it in the root directory

Reply:
Moved to the root directory.